modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/osb-site-initializer-partner-portal-test/src/testFunctional/macros/PRMMDFRequest.macro

```diff
@@ -80,6 +80,43 @@ definition {
 		Alert.viewSuccessMessage();
 	}
 
+	@summary = "Verify that is possible edit MDF activity "
+	macro editMDFActivity {
+		Click(locator1 = "PRMMDFRequest#EDIT_ACTIVITY_BUTTON");
+
+		if (!(isSet(activitiyIndex))) {
+			var activityIndex = 1;
```

Comment:
Hi @MGabriellaS, the `activityIndex` should be = 0 here, since the PRM indexes starts on 0.